core/src/main/java/org/jdbi/v3/core/argument/BuiltInArgumentFactory.java

```diff
@@ -86,6 +88,8 @@
         register(map, double.class, Types.DOUBLE, PreparedStatement::setDouble);
         register(map, Float.class, Types.FLOAT, PreparedStatement::setFloat);
         register(map, float.class, Types.FLOAT, PreparedStatement::setFloat);
+        register(map, Inet4Address.class, Types.OTHER, (p, i, v) -> p.setObject(i, v.getHostAddress(), Types.OTHER));
+        register(map, Inet6Address.class, Types.OTHER, (p, i, v) -> p.setObject(i, v.getHostAddress(), Types.OTHER));
```

Comment:
I'm curious why these are separate, considering the two factories are identical. Couldn't there be just one arg factory for `InetAddress`?